.circleci/config.yml

```diff
@@ -74,3 +86,6 @@ workflows:
           filters:
             branches:
               only: master
+      - github_release:
+          requires:
+            - production_release
```

Comment:
nitpick: noticed that the name for this step is `release` in other repos & it is `production_release` on this one. 

Worth changing, to keep things in line.

Otherwise 👍

Reply:
Done